examples/release_highlights/plot_release_highlights_0_22_0.py

```diff
@@ -149,3 +149,43 @@
     # recomputed.
     estimator.set_params(isomap__n_neighbors=5)
     estimator.fit(X)
+
+############################################################################
+# Stacking Classifier (and Regressor)
+# -----------------------------------
+# :class:`~sklearn.ensemble.StackingClassifier` and
+# :class:`~sklearn.ensemble.StackingRegressor`
+# allow you to have a stack of estimators with a final classifier or
+# a regressor.
+# Stacked generalization consists in stacking the output of individual
+# estimator and use a classifier to compute the final prediction. Stacking
+# allows to use the strength of each individual estimator by using their output
+# as input of a final estimator.
+# Note that ``estimators_`` are fitted on the full ``X`` while
+# ``final_estimator_`` is trained using cross-validated predictions of the
```

Comment:
I would just use the term "base estimators" and "final estimator" instead of using the attribute names.

Also maybe remove "Note that"